utils/mosh/Makefile

```diff
@@ -0,0 +1,116 @@
+#
+# Copyright (C) 2018 OpenWrt.org
```

Comment:
please either drop this or use your own name, you cannot assign copyright like this.

Reply:
https://github.com/dragino/openwrt-packages/blob/master/CONTRIBUTING.md says:
"An up-to-date copyright notice. Use OpenWrt if no other present or supply your own."
Anyway, I changed it.

Reply:
Dragino's fork of packages isn't in definitive.  See https://github.com/openwrt/packages/blob/master/CONTRIBUTING.md#makefile-contents-should-contain instead.  (it reflected what was practice when it was forked presumably, but not anything present)